libraries/from-bodyxml/index.js

```diff
@@ -47,7 +47,7 @@ function toValidLayoutWidth(layoutWidth) {
 
 /**
  * @template {UNode | UParent} Node
- * @typedef {Omit<Node, "children"> & (Node extends UParent ? {children?: Node["children"]} : {children?: null})} TransNode
+ * @typedef {Omit<Node, "children"> & (Node extends UParent ? {children?: Node["children"]} : {children: null})} TransNode
```

Comment:
This fixes the type hinting in this file to error if a childless transformer does _not_ have `children: null`